common/djangoapps/pipeline_mako/tests/test_render.py

```diff
@@ -7,20 +7,20 @@
 class RequireJSPathOverridesTest(TestCase):
     """Test RequireJS path overrides. """
 
-    OVERRIDES = [
-        'js/vendor/jquery.min.js',
-        'js/vendor/backbone-min.js',
-        'js/vendor/text.js'
-    ]
+    OVERRIDES = {
+        'jquery': 'js/vendor/jquery.min.js',
+        'backbone': 'js/vendor/backbone-min.js',
+        'text': 'js/vendor/text.js'
+    }
 
     OVERRIDES_JS = [
         "<script type=\"text/javascript\">",
         "(function (require) {",
         "require.config({",
         "paths: {",
-        "'js/vendor/jquery.min': 'js/vendor/jquery.min',",
-        "'js/vendor/backbone-min': 'js/vendor/backbone-min',",
-        "'js/vendor/text': 'js/vendor/text'",
+        "'jquery': 'js/vendor/jquery.min',",
+        "'text': 'js/vendor/text',",
+        "'backbone': 'js/vendor/backbone-min'",
```

Comment:
Missing comma before empty quotes? I haven't looked at this PR enough to really understand what this is about.